src/test/java/com/artipie/vertx/VertxSliceServerTest.java

```diff
@@ -139,6 +139,26 @@ public void basicGetRequest() {
         MatcherAssert.assertThat(actual, Matchers.equalTo(expected));
     }
 
+    @Test
+    public void basicGetRequestCL() {
+        final String expected = "Hello World!!!";
+        this.start(
+            (line, headers, body) -> connection -> connection.accept(
+                RsStatus.OK,
+                new Headers.From(
+                    "Content-Length",
+                    Integer.toString(expected.length())
+                ),
+                Flowable.fromArray(ByteBuffer.wrap(expected.getBytes()))
+            )
+        );
+        final String actual = this.client.get(this.port, VertxSliceServerTest.HOST, "/hello2")
+            .rxSend()
+            .blockingGet()
+            .bodyAsString();
+        MatcherAssert.assertThat(actual, Matchers.equalTo(expected));
```

Comment:
@Sammers21 the test should check that there is not `Transfer-Encoding` header in response, otherwise it does not really check the issue at hand